src/store/stubbings.coffee

```diff
@@ -42,15 +42,24 @@ invokeCallbackFor = (stubbing, actualArgs) ->
   return unless _.some(stubbing.args, callback.isCallback)
   _.each stubbing.args, (expectedArg, i) ->
     return unless callback.isCallback(expectedArg)
-    callbackArgs = if expectedArg.args?
-      expectedArg.args
-    else if stubbing.config.plan == 'thenCallback'
-      stubbing.stubbedValues
-    else
-      []
+    args = callbackArgs stubbing, expectedArg
```

Comment:
if you wouldn't mind, all the single-line function invocations use `()`s in this project